### PR TITLE
feat: disconnect observer while fetching

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Add a level of security to prevent multiple fetching in case the Infinite Scroll is used on a slow network.

# Changes

- disconnect observer before dispatching event

### CODE REVIEW

File: frontend/svelte/src/lib/components/ui/InfiniteScroll.svelte [hunk 1]
```diff
@@ -38,6 +41,8 @@
 
   afterUpdate(() => {
     // The DOM has been updated. We reset the observer to the current last HTML element of the infinite list.
+
+    // We disconnect previous observer first. We want to avoid to obsevere multiple elements.
```
Comment (nmattia) [inline frontend/svelte/src/lib/components/ui/InfiniteScroll.svelte:45]: Something fishy in that last sentence

Comment (peterpeterparker) [reply frontend/svelte/src/lib/components/ui/InfiniteScroll.svelte:48]: Done

**Comment by mstrasinskis** [comment]
Just one thing -  `const` would be preferable here `let observer: IntersectionObserver`

**Comment by peterpeterparker** [comment]
@mstrasinskis const typo fixed.

**Review by mstrasinskis — APPROVED** [review]
Thank you!

**Pull request merged**